Gemfile.lock

```diff
@@ -1,6 +1,7 @@
 GEM
   remote: https://rubygems.org/
   specs:
+<<<<<<< HEAD
```

Comment:
You committed unmerged code.  Best bet when there is a conflict in your Gemfile.lock is to delete it and run bundle install again.